prow/test/integration/config/nginx.yaml

```diff
@@ -1,3 +1,14 @@
+# Pin the ingress-nginx manifest to fb72fcd81772fb3eca923897aec2d92fa5bdff41 (Release 1.1.2) on 3/18/2022.
```

Comment:
in https://github.com/kubernetes/test-infra/pull/29496 I only reverted the image change, not sure if we want to revert both?